SWBF2Admin/Web/WebServer.cs

```diff
@@ -237,7 +237,7 @@ private WebUser CheckAuth(HttpListenerContext ctx)
 
             if (authCache.TryGetValue(identity.Name, out WebUser user))
             {
-                if (!Util.Md5(identity.Password).Equals(user.PasswordHash))
+                if (!Util.Sha256(identity.Password).Equals(user.PasswordHash))

```

Comment:
This is a backwards-incompatible change

Reply:
I agree with @yonilerner; just replacing Md5 with Sha256 would prevent users from logging in when updating the binary.
7f9aa760f8dcf46f96ae08e9f2b26ce2bdaa54f6 substitutes md5 with salted pbkdf2 while keeping compatibility with the existing database.